api/report_server.thrift

```diff
@@ -298,6 +305,30 @@ service codeCheckerDBAccess {
                                              4: ReportFilterList reportFilters)
                                              throws (1: shared.RequestFailed requestError),
 
+  // The Counter value should be converted beck to the
+  // proper Severity type by the client.
+  Counters getSeverityCounts(1: list<i64> runIds,
```

Comment:
As I've already mentioned before I do not wanted to create a separate Counter type where the value and the count is an integer too. There are 5 api calls right now and only two of them return enums as value.
A CounterStr and a CounterInt would be better? @whisperity ?